ext/bcmath/libbcmath/src/init.c

```diff
@@ -32,22 +32,20 @@
 #include "bcmath.h"
 #include <stdbool.h>
 #include <stddef.h>
+#include <string.h>
 #include "zend_alloc.h"
 
 /* new_num allocates a number and sets fields to known values. */
 bc_num _bc_new_num_ex(size_t length, size_t scale, bool persistent)
 {
-	bc_num temp;
-	/* PHP Change: malloc() -> pemalloc(), removed free_list code */
```

Comment:
I don't think those 2 comments are really harmful, might need to be corrected to mention ``safe_`` in front

Reply:
Reverted comments